aeron-cluster/src/main/java/io/aeron/cluster/ConsensusModuleAgent.java

```diff
@@ -2546,7 +2548,9 @@ LogReplication newLogReplication(
             logRecordingId,
             ctx.archiveContext().controlRequestStreamId(),
             leaderArchiveEndpoint,
-            stopPosition);
+            stopPosition,
+            LOG_REPLICATION_PROGRESS_CHECK_TIMEOUT_NS,
```

Comment:
Should these be members of the Election class?

Reply:
I was going to add configuration options for these, but instead I've reused the leader heartbeat interval/timeout.